data/script_cmd_table.inc

```diff
@@ -229,6 +229,9 @@ gScriptCmdTable::
 	.4byte ScrCmd_bufferitemnameplural              @ 0xe2
 	.4byte ScrCmd_dynmultichoice                    @ 0xe3
 	.4byte ScrCmd_dynmultipush                      @ 0xe4
+    .4byte ScrCmd_getdifficulty                     @ 0xe5
+    .4byte ScrCmd_setdifficulty                     @ 0xe6
+
```

Comment:
TAB vs. space, probably don't need the trailing newline

Reply:
removed!
[`b178d44` (#5337)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5337/commits/b178d44e6c9113ef3d43c2b6b1462a3598d22ca1)